plugins/sqlite_dataset.cc

```diff
@@ -627,13 +631,13 @@ struct SqliteSparseDataset::Itl
         doCommand("CREATE TABLE IF NOT EXISTS rows ("
                   "  rowNum INTEGER PRIMARY KEY, "
                   "  rowHash INT NOT NULL, "
-                  "  rowName INT NOT NULL "
+                  "  rowName TEXT NOT NULL "
                   ")");
 
         doCommand("CREATE TABLE IF NOT EXISTS cols ("
                   "  colNum INTEGER PRIMARY KEY, "
                   "  colHash INT NOT NULL, "
-                  "  colName INT NOT NULL "
+                  "  colName TEXT NOT NULL "
```

Comment:
Is that really realted to Ubuntu 16.04?

Reply:
Yes, it fixes something that worked on 14.04 but no longer does on 16.04.